admin/metabox/class-metabox.php

```diff
@@ -1154,6 +1155,23 @@ private function get_custom_fields_replace_vars( $post ) {
 		return $custom_replace_vars;
 	}
 
+	/**
+	 * Checks if we're past the date set for the Jetpack Boost ad.
+	 *
+	 * @return bool Whether the Jetpack Boost ad must be shown.
+	 */
+	private function checkJetpackBoostAdDate() {
+		$now                          = new DateTime( 'now' );
+		$jetpack_boost_ad_date_string = WPSEO_Options::get( 'jetpack_ad_start_date' );
```

Comment:
* What if the option does not exist? Perhaps a second parameter for safety?
* Also, if this is the default, it will be an empty string in your current implementation. What happens on the next line then?

Reply:
I'm using `1970-01-01 00:00:00` to be consistent with the default value.